### PR TITLE
[Term Entry] Python Math Module : .remainder()

### PR DESCRIPTION
**Pull request by Piergiulio**
…ted it

### Description

- Add the math.remainder() term to Python
- Closes issue #4349 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/python/concepts/math-module/terms/math-remainder/math-remainder.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (avdhoottt) [inline content/python/concepts/math-module/terms/math-remainder/math-remainder.md:110]:
```suggestion
```

Comment (Piergiulio) [reply content/python/concepts/math-module/terms/math-remainder/math-remainder.md:110]: @avdhoottt Thank you very much I will correct it straight away

**Comment by Piergiulio** [comment]
@avdhoottt following the suggested changes I committed and pushed the changes on the file remainder.md

**Comment by Piergiulio** [comment]
Good evening @avdhoottt , I've corrected remainder.md with all the suggested changes. The only thing I'm not sure of is the Note on lines 28-29 . Would that be alright ? Thank you very much for the advices

File: content/python/concepts/math-module/terms/math-remainder/math-remainder.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+In this example we will return the remainder between two **Integer** values.
+
+```python
+# Import math Library
+import math
+
+# Return the remainder of x/y
+print(math.remainder(10, 2))
+print(math.remainder(34, 3))
+```
```
[the remark this answers is not in the document]
Comment (Piergiulio) [reply content/python/concepts/math-module/terms/math-remainder/math-remainder.md:42]: @avdhoottt I have done all the changes requested on the file math-remainder.md and committed it , I'm now waiting for the next step

**Review by Piergiulio — COMMENTED** [review]
@avdhoottt I have done all the changes requested on the file math-remainder.md and committed it , I'm now waiting for the next step

**Comment by Piergiulio** [comment]
@avdhoottt as it has passed quite some time from the last review, just checking if there is something I'm doing wrong with my corrections on the file or something I have to do in the pull request which is missing.

File: content/python/concepts/math-module/terms/math-remainder/math-remainder.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+```pseudo
+math.remainder(x,y)
+```
+
+  - `x`: It is the number to be divided.
+  - `y`: It is the number by which `x` is divided.
+
+>**Note:** <br>
+>The Parameter `y` has to be a non-zero number or will return a **`ValueError`**
```
Comment (avdhoottt) [inline content/python/concepts/math-module/terms/math-remainder/math-remainder.md:29]:
```suggestion
>**Note:** The Parameter `y` has to be a non-zero number or will return a `ValueError`.
```

File: content/python/concepts/math-module/terms/math-remainder/math-remainder.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+```python
+# Import math Library
+import math
+
+# Return the remainder of x/y
+print(math.remainder(10, 2))
+print(math.remainder(34, 3))
+```
+
+The `output` for the above line of code will be:
```
Comment (avdhoottt) [inline content/python/concepts/math-module/terms/math-remainder/math-remainder.md:44]:
```suggestion
The output for the above line of code will be:
```

File: content/python/concepts/math-module/terms/math-remainder/math-remainder.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+
+```python
+# Import math Library
+import math
+
+print(math.remainder(25.5, 5.7))
+print(math.remainder(58.9, 5.5))
+```
+
+the `output` for the above line of code will be:
```
Comment (avdhoottt) [inline content/python/concepts/math-module/terms/math-remainder/math-remainder.md:63]:
```suggestion
the output for the above line of code will be:
```

File: content/python/concepts/math-module/terms/math-remainder/math-remainder.md [hunk 6]
```diff
@@ -0,0 +1,101 @@
...
+
+```python
+# Import math Library
+import math
+
+print(math.floor(math.remainder(25.5, 5.7)))
+print(math.ceil(math.remainder(25.5, 5.7)))
+```
+
+The `output` for the above line of code will be:
```
Comment (avdhoottt) [inline content/python/concepts/math-module/terms/math-remainder/math-remainder.md:82]:
```suggestion
The output for the above line of code will be:
```

File: content/python/concepts/math-module/terms/math-remainder/math-remainder.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+```python
+# Import math Library
+import math
+
+print(math.floor(math.remainder(25.5, 5.7)))
+print(math.ceil(math.remainder(25.5, 5.7)))
+```
+
+The output for the above line of code will be:
```
Comment (avdhoottt) [inline content/python/concepts/math-module/terms/math-remainder/math-remainder.md:81]:
```suggestion
The output for the above code will be:
```

**Review by avdhoottt — APPROVED** [review]
@Piergiulio Apologies for the delay. It looks good for the second review,

File: content/python/concepts/math-module/terms/math-remainder/math-remainder.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+```
+
+  - `x`: It is the number to be divided.
+  - `y`: It is the number by which `x` is divided.
+
+>**Note:** The Parameter `y` has to be a non-zero number or will return a `ValueError`.
+
+## Example 1
+
+This example will return the remainder between two **Integer** values.
```
Comment (ishg-153) [inline content/python/concepts/math-module/terms/math-remainder/math-remainder.md:32]:
```suggestion
This example will return the remainder between two Integer values.
```

File: content/python/concepts/math-module/terms/math-remainder/math-remainder.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+The output for the above code will be:
+
+```shell
+0.0
+1.0
+```
+
+## Example 2
+
+This example will return the remainder between two **Float** values.
```
Comment (ishg-153) [inline content/python/concepts/math-module/terms/math-remainder/math-remainder.md:52]:
```suggestion
This example will return the remainder between two Float values.
```

File: content/python/concepts/math-module/terms/math-remainder/math-remainder.md [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+
+```python
+# Import math Library
+import math
+
+print(math.remainder(25.5, 5.7))
+print(math.remainder(58.9, 5.5))
+```
+
+the output for the above code will be:
```
Comment (ishg-153) [inline content/python/concepts/math-module/terms/math-remainder/math-remainder.md:62]:
```suggestion
The output for the above code will be:
```

File: content/python/concepts/math-module/terms/math-remainder/math-remainder.md [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+
+```shell
+2.6999999999999993
+-1.6000000000000014
+```
+
+## Example 3
+
+The use of multiple `Methods` can then convert the result to an **Integer** value.
```
Comment (ishg-153) [inline content/python/concepts/math-module/terms/math-remainder/math-remainder.md:71]:
```suggestion
The use of multiple `Methods` can then convert the result to an Integer value.
```

File: content/python/concepts/math-module/terms/math-remainder/math-remainder.md [hunk 12]
```diff
@@ -0,0 +1,100 @@
...
+The **`math.remainder()`** method is the remainder of the division `x/y`. It will return a float value, representing the remainder. The remainder is the difference between `x` and the closest integer multiple of `y`.
+
+## Syntax
+
+```pseudo
+math.remainder(x,y)
+```
+
+  - `x`: It is the number to be divided.
+  - `y`: It is the number by which `x` is divided.
```
Comment (ishg-153) [inline content/python/concepts/math-module/terms/math-remainder/math-remainder.md:26]:
```suggestion
  - `y`: It is the number by which `x` is divided by.
```

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@Piergiulio I have requested some changes. Please make those, let me know once done.

**Comment by Piergiulio** [comment]
Good evening @ishg-153  I've made the changes suggested to remainder.md 
let me know if everything is fine, thanks for the good advices 
Piergiulio

**Review by ishg-153 — APPROVED** [review]
@Piergiulio made some changes from my end. This is ready to merge!

**Pull request merged**